components/StreamField/blocks/TopicsBlock/TopicsBlock.module.scss

```diff
@@ -1,9 +1,10 @@
 .grid {
     display: grid;
 
-    @include media-query(large) {
+    @include media-query(medium) {
         grid-template-columns: repeat(4, 1fr);
-        margin: 0 -10rem;
+        column-gap: 1%;
```

Comment:
**Question**: This is cool, I don't think I've ever seen a % used for this before. I'm guessing this is relative to the grid container width (not height), making the spacing of the items responsive on resize?